docs/user-guide/global-configurations/tags-policy.md

```diff
@@ -1,51 +1,165 @@
-# Tags
+# Tags Policy
 
 ## Introduction [![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/elements/EnterpriseTag.svg)](https://devtron.ai/pricing)
 
-Devtron's `Tags Policy` feature enables you to assign tags to your applications. Devtron also offers the option to propagate the tags assigned to an application as labels within the associated Kubernetes resources. 
+Tags are key-value pairs containing metadata that helps in identifying and categorizing Devtron apps. For example, you can have a tag named *`Business-Unit`* with a value assigned during application creation (or later, if mandatory).
 
-## Add Tags
+The **Tags Policy** feature lets you create tags for applications, and enforce deployment-specific rules if needed. It also allows you to propagate these tags as labels to the Kubernetes resources associated with the application.
```

Comment:
Then mention With tag policy ~lets you create tags for applications~ you can enforce that a tag must be provided before application creation or before deployment to an environment.